e2e/cypress/support/api/user.js

```diff
@@ -136,6 +142,38 @@ Cypress.Commands.add('apiCreateAdmin', () => {
     });
 });
 
+Cypress.Commands.add('apiCreateCustomAdmin', (username) => {
+    const {password} = getAdminAccount();
+    const sysadminUser = {
+        username,
+        password,
+        first_name: 'Kenneth',
+        last_name: 'Moreno',
+        email: username + '@sample.mattermost.com',
+    };
+
+    const options = {
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        method: 'POST',
+        url: '/api/v4/users',
+        body: sysadminUser,
+    };
+
+    // # Create a new user
+    return cy.request(options).then((res) => {
+        expect(res.status).to.equal(201);
+        return cy.request({
+            headers: {'X-Requested-With': 'XMLHttpRequest'},
+            method: 'PUT',
+            url: `/api/v4/users/${res.body.id}/roles`,
+            body: {roles: 'system_admin system_user'},
+        }).then((res2) => {
+            expect(res2.status).to.equal(200);
+            return cy.wrap({sysadmin: res.body});
+        });
+    });
```

Comment:
We already have api commands for creating a user and patching user roles, so this can be:
```javascript
    const sysadminUser = generateRandomUser('other-admin');

    return cy.apiCreateUser({user: sysadminUser}).then(({user}) => {
        return cy.apiPatchUserRoles(user.id, ['system_admin', 'system_user']).then(() => {
            return cy.wrap({sysadmin: user});
        });
    });
```